src/components/ModalCard.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import {
+  Typography, withStyles, Modal, Slide, Card, CardMedia, Avatar, CardContent,
+} from "@material-ui/core";
+import { bigStyles } from "screens/DashboardStyles";
+
+function ModalCard(props) {
+  const {
+    classes, modalData, openModal, onClose,
+  } = props;
+  const { birth_date, phone_number } = modalData[1];
+
+  return (
+    <div>
+      <Modal
+        className="flexCenter"
+        open={openModal}
+        onClose={onClose}
+      >
+        <Slide
+          direction="up"
+          in={openModal}
+          mountOnEnter
+          unmountOnExit
+        >
+          <Card
+            className={classes.card}
+          >
+            <CardMedia
+              className={classes.media}
+            >
+              <Avatar
+                className={classes.innerAvatar}
+                src={`https://randomuser.me/api/portraits/men/${Math.floor(Math.random() * 100)}.jpg`}
+                alt="avatar placeholder"
+              />
+            </CardMedia>
+            <CardContent>
+              <Typography variant="h5">{modalData[0]}</Typography>
```

Comment:
what is 0 ?

Reply:
It's `props.value` from line 94 in `Dashboard.js`